lib/heuristics/scheduling_heuristic.rb

```diff
@@ -397,7 +397,7 @@ def reorder_stops(vrp)
         all_routes.each{ |day, route_data|
           next if route_data[:stops].uniq{ |s| s[:point_id] }.size <= 1
 
-          corresponding_vehicle = vrp.vehicles.find{ |v| v.id == "#{vehicle_id}_#{day}" }
+          corresponding_vehicle = vrp.vehicles.find{ |v| v.id.match?(/#{vehicle_id}.*_#{day}/) }
```

Comment:
Can't see why we need such a modification in this PR :thinking: 

Expansion of shipments into services shouldn't require a modification of a logic dependent on vehicle_ids, no?

Reply:
Indeed, this should not be needded now that we removed original_id in test_helper -> coerce